src/pages/commits/ActiveCommits/ProjectHealth.stories.tsx

```diff
@@ -22,80 +41,131 @@ export const ActualWaterfallPage = ({
   hasFilters,
 }) => {
   const updatedVersions = versions.map((version) =>
-    formatVersion(version, buildVariantCount, taskCount)
+    formatVersion(version, buildVariantCount, taskCount, hasTaskFilter)
```

Comment:
Is it possible to tweak this function so that there isn't whitespace when `hasTaskFilter` is `false`?

<img width="188" alt="Screen Shot 2022-12-22 at 1 02 22 PM" src="https://user-images.githubusercontent.com/47064971/209197679-5491d5bd-12a6-4e36-bbac-39c7f355c92c.png">

Reply:
I tried but couldn't figure out an elegant solution. :/

Reply:
that's okay, I kind of hope the design changes on this page anyways (Like with the build variants listed on the left side like Evergreen), which would help reduce the complexity in the story !